CacheService/docker_context/LegacyStockServerHandler.py

```diff
@@ -17,6 +25,26 @@ async def quoteSocket(self):
         return reader, writer
 
     async def getQuote(self, trans_num, user_id, stock_symbol):
+        if stock_symbol in self.tracker.keys():
+            await self.tracker[stock_symbol]
```

Comment:
won't this just rerun the task? shouldn't we check to see if the task is done or not

Reply:
Old ways of thought die hard.